WooCommerce/Classes/ViewRelated/Orders/Simple Payments/Summary/SimplePaymentsSummaryViewModel.swift

```diff
@@ -149,9 +196,18 @@ final class SimplePaymentsSummaryViewModel: ObservableObject {
                      presentNoticeSubject: PassthroughSubject<SimplePaymentsNotice, Never> = PassthroughSubject(),
                      currencyFormatter: CurrencyFormatter = CurrencyFormatter(currencySettings: ServiceLocator.currencySettings),
                      stores: StoresManager = ServiceLocator.stores) {
+
+        // Generate `TaxLine`s to represent `taxes` inside `View`.
+        let taxLines = order.taxes.map({
+            TaxLine(id: $0.taxID,
+                    title: "\($0.label) (\($0.ratePercent)%)",
+                    value: currencyFormatter.formatAmount($0.totalTax) ?? $0.totalTax)
```

Comment:
Nit: This logic can be simplified to be separate from `SimplePaymentsSummaryViewModel` in two ways:
- Use a convenience initializer for TaxLine that accepts `OrderTaxLine` as input
- Replace TaxLine with just an extension for OrderTaxLine which calculates the displayed title and value and conforms to Identifiable.

The separation of logic also makes it easier to test the computed variables too I think.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/5943/commits/6dfb8324c504c879786ef80e24a3cfbded5e0b0b